### PR TITLE
Add case studies page

### LINKED ISSUE #12: Add "Case Studies" page
Add a "Case Studies" page - designs and copy to follow.

There should be a link to this page from a drop down under the "Hire Page" in the menu.

### PR DESCRIPTION
**Pull request by ChillFish8**
Closes #12

### CODE REVIEW

File: _sass/pages/_case-studies.scss [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    .opening-section > div {
+      width: 100%;
+      margin-top: 2rem;
+    }
+  }
+
+  .main-container {
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
```
Comment (thehereward) [inline _sass/pages/_case-studies.scss:129]: This is just `px-4`

File: case-studies.html [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+            y.parentNode.insertBefore(t, y);
+        })(window, document, "clarity", "script", "a532331rj3");
+    </script>
+
+    <link href="styles.css" rel="stylesheet" type="text/css"/>
+    <script async src="/console-message.js"></script>
+
+    <title>Making An Impact | TechSwitch</title>
+
+    {% include analytics.html %}
```
[the remark this answers is not in the document]
Comment (sasha-burgoyne) [reply case-studies.html:50]: Tragically yes - three were fairly recently added by the marketing team for the various different things they're doing in their campaign

File: case-studies.html [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+                <polygon class="triangle" points="60 0,0 60,60 60" stroke-width="2"/>
+            </svg>
+            <svg fill="#ffc100" height="60"
+                 style="position: absolute;top: 0; left: 0;margin-left: 0.5rem;margin-top:  0.5rem;" width="60">
+                <polygon class="triangle" points="60 0,0 60,60 60" stroke-width="2"/>
+            </svg>
+        </div>
+    </div>
+
+    <!-- Separator -->
```
Comment (thehereward) [inline case-studies.html:127]:
```suggestion
```

File: case-studies.html [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+                    Anyone who’s travelled on the UK’s rail network since it was privatised,
+                    may well have been on an Angel Trains carriage.<br/><br/>
+                    The business owns and maintains more than a quarter of all UK passenger rolling stock,
+                    which amounts to well over 4,000 vehicles. It leases these to train operating
+                    companies including Avanti West Coast, ScotRail and CrossCountry.<br/><br/>
+
+                    Angel Trains has invested more than £5 billion in its trains since privatisation,
+                    and carries out 1,000+ heavy maintenance overhauls and component changes each year.
+                </p>
```
Comment (thehereward) [inline case-studies.html:151]: This should be multiple `<p>` tags rather than using `<br />`.

File: case-studies.html [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+                    another a materials scientist, and a third had managed an international IT team.
+                    We would genuinely have been happy hiring any of them.”<br/><br/>
+
+                    “One of our objectives when hiring for this role was to ensure that we had a diverse
+                    range of candidates and it was great to see that delivered.”
+                </p>
+            </div>
+            <img
+                    alt="One of our objectives when hiring for this role was to ensure that we had a diverse
+                    range of candidates and it was great to see that delivered."
```
Comment (thehereward) [inline case-studies.html:216]: Add the attribution to the alt text.

**Review by thehereward — APPROVED** [review]

**Pull request merged**